modules/form-builder/module.json

```diff
@@ -1,15 +1,12 @@
 {
   "dependencies": [
     "component",
+    "event-emitter",
+    "form",
     "map",
-    "util",
+    "picker",
     "selection",
-    "event-emitter",
     "tag-input",
-    "picker"
-  ],
-  "keywords": [
-    "form",
-    "builder"
+    "util"
```

Comment:
Remove 'keywords' and sort dependencies alphabetically - done this in every module.json I've looked in because I figure it's a reasonable thing to do :)